Description: The qmltime should be a tool, not a normal executable or an app.
Origin: upstream, https://code.qt.io/cgit/qt/qtdeclarative.git/commit/?h=5.15&id=30c7a6c6a8742648
Last-Update: 2021-01-18

--- a/tools/qmltime/qmltime.pro
+++ b/tools/qmltime/qmltime.pro
@@ -1,4 +1,3 @@
-TEMPLATE = app
 TARGET = qmltime
 QT += qml quick
 QT += quick-private
@@ -12,3 +11,5 @@ QMAKE_TARGET_DESCRIPTION = QML Time
 
 SOURCES += qmltime.cpp
 HEADERS += qmltime.h
+
+load(qt_tool)
